demo/summer-2016.html

```diff
@@ -215,10 +216,10 @@
 <script src="http://maxcdn.bootstrapcdn.com/bootstrap/2.3.2/js/bootstrap.min.js"></script>
 
 <script>
+// var editor;
 require.config({ paths : { "ace" : "../lib/ace" } });
-require(["ace/ace", "ace/ext/language_tools", "ace/ext/papeeria/katex-previewer", "ace/ext/papeeria/highlighter"], 
-  function(ace, langTools, katexPreviewer, highlighter) {
-    var editor = ace.edit("editor");
+require(["ace/ace", "ace/ext/language_tools", "ace/ext/papeeria/context-previewer", "ace/ext/papeeria/highlighter"], function(ace, langTools, katexPreviewer, highlighter) {
+    editor = ace.edit("editor");
```

Comment:
Well, for development purposes you can do whatever makes sense, but in production code it is better to be safe, even if this particular file is not going to production. One global variable assigmnent here, another read access there which for some reasons slipped through the review, and finally you have code which heavily depends on global variables.